radix.py

```diff
@@ -0,0 +1,108 @@
+def radixsort(input_list):
+    """Sort list of integers and strings with radix sort algorithm."""
+    if type(input_list[0]) is str:
+        # for case when sorting string
+        base = 256
+        string = True
+        place = 0
```

Comment:
Interesting!  so you are doing both sorts in one algorithm.  I'm curious how this will work out for you!